charts/csm/Chart.yaml

```diff
@@ -0,0 +1,75 @@
+#
+#
+# Copyright © 2023 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+apiVersion: v2
+name: csm
```

Comment:
During another discussion about naming conventions, we can expand the CSM acronym to be "container-storage-modules" to avoid confusion with other Dell CSM product.

Reply:
Sure, will update the name in the file and will update the chart name to "container-storage-modules"